README.md

```diff
@@ -1,52 +1,49 @@
 [![CLOMonitor](https://img.shields.io/endpoint?url=https://clomonitor.io/api/projects/cncf/kubeflow/badge)](https://clomonitor.io/projects/cncf/kubeflow)
+[![OpenSSF Best Practices](https://www.bestpractices.dev/projects/2660/badge)](https://www.bestpractices.dev/projects/2660)
 
-The Kubeflow makes AI/ML on Kubernetes simple, portable and scalable.
+<img src="./logo/icon.svg" width="120">
 
----
+## About Kubeflow
 
-## Documentation
+[Kubeflow](https://www.kubeflow.org/) makes artificial intelligence and machine learning simple, portable, and scalable.
+We are an _ecosystem_ of [Kubernetes](https://kubernetes.io/) based components for each stage in the [AI/ML Lifecycle](https://www.kubeflow.org/docs/started/architecture/#kubeflow-components-in-the-ml-lifecycle) with support for best-in-class open source [tools and frameworks](https://www.kubeflow.org/docs/started/architecture/#kubeflow-ecosystem).
 
-Please refer to the official docs at [kubeflow.org](http://kubeflow.org).
+Please refer to the official [documentation](https://www.kubeflow.org/docs/) for more information.
 
 ## Kubeflow Components
 
-The Kubeflow ecosystem is composed of multiple open-source projects for each stage in
-[the ML lifecycle](https://www.kubeflow.org/docs/started/architecture/#kubeflow-components-in-the-ml-lifecycle).
+The [Kubeflow Ecosystem](https://www.kubeflow.org/docs/started/architecture/#kubeflow-ecosystem) is composed of several projects known as [Kubeflow Components](https://www.kubeflow.org/docs/components/).
 
-Learn more about each project in [the Kubeflow documentation](https://www.kubeflow.org/docs/components/).
+The following table lists the components and their respective source code repositories:
 
-Please use the following GitHub repositories to open issues and pull requests for
-[the different Kubeflow components](https://www.kubeflow.org/docs/started/introduction/#what-are-standalone-kubeflow-components):
+| Component                                                                           | Source Code                                                                   |
+|-------------------------------------------------------------------------------------|-------------------------------------------------------------------------------|
+| [KServe](https://www.kubeflow.org/docs/external-add-ons/kserve/)                    | [`kserve/kserve`](https://github.com/kserve/kserve)                           |
+| [Kubeflow Katib](https://www.kubeflow.org/docs/components/katib/)                   | [`kubeflow/katib`](https://github.com/kubeflow/katib)                         |
+| [Kubeflow Model Registry](https://www.kubeflow.org/docs/components/model-registry/) | [`kubeflow/model-registry`](https://github.com/kubeflow/model-registry)       |
+| Kubeflow MPI Operator                                                               | [`kubeflow/mpi-operator`](https://github.com/kubeflow/mpi-operator)           |
```

Comment:
I think, you can link this page here for MPI: https://www.kubeflow.org/docs/components/training/user-guides/mpi/

Reply:
We probably need to create a separate "MPI Operator" section on the website docs, because that page does not really explain what it is?

I can add the link as long as we plan to redirect it in the future.

Reply:
In the future, we can discuss about it, since we are in the transition period for MPI V2 to TrainJob V2.
Let's add the MPIJob page for now.
Any objections @kubeflow/wg-training-leads @alculquicondor ?

Reply:
@alculquicondor This is needed to re-direct users to the MPI Operator documentation from [the `kubeflow/kubeflow` README](https://github.com/kubeflow/kubeflow?tab=readme-ov-file#kubeflow-components).
I suggested, that we will link this doc for MPI Operator for now: https://www.kubeflow.org/docs/components/training/user-guides/mpi/

Reply:
That seems reasonable